sorc/bufr_dupcor.fd/dupcor.f

```diff
@@ -578,6 +578,13 @@ PROGRAM BUFR_DUPCOR
       CALL UFBTAB(-LUBFI,UFBTAB_8,1,1,MXTB,' ')
       CALL OPENBF(0,'QUIET',0) ! return to default wrt degree of print
 
+      IF(MXTB.EQ.0) THEN
+         PRINT *
+         PRINT *, '### WARNING: A total of ZERO input aircraft reports'
```

Comment:
Suggest:
PRINT *, '### WARNING: A total of ZERO input reports //dupcor'

Reply:
did it